### DISCUSSION TITLE
rebuilding css in php is too slow and sometimes it doesn't build when too many classes are built, i use cli install

### DISCUSSION THREAD

**Discussion by altamisatmaja** [tailwindlabs/tailwindcss#13030] [Help]
TailwindCSS v3.4.1 and i use PHP MVC 🗡️ 

**What version of Node.js are you using?**

For example: v18.0.0

**What browser are you using?**

I use chrome

**What operating system are you using?**

I use MacOs

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

A reproduction is **required** when filing an issue — any issue opened without a reproduction will be closed and you'll be asked to create a new issue that includes a reproduction. We're a small team and we can't keep up with the volume of issues we receive if we need to reproduce each issue from scratch ourselves.

**Describe your issue**

Rebuilding CSS in PHP using a CLI installation can often result in slow performance, particularly when there are a large number of classes being built. Additionally, there are instances where the rebuilding process fails, especially under heavy loads or when dealing with an extensive set of classes. To mitigate these issues, I've opted to utilize a CDN for Tailwind CSS across all projects. This approach not only reduces the server load but also ensures faster loading times for web pages

**Comment by reinink** [comment]
Hey! Converting this to a discussion as it doesn't include a minimal reproduction. If you can put together a simple reproduction that clearly illustrates this issue, we'd be happy to have a look.

Also, just a note, the [Play CDN](https://tailwindcss.com/docs/installation/play-cdn) is designed for development purposes only, and is not designed for use in production. We highly recommend building your CSS prior to deployment for best results.
